lib/grpperm.gi

```diff
@@ -466,6 +466,48 @@ InstallMethod( NrMovedPoints,
     G -> NrMovedPoints( GeneratorsOfGroup( G ) ) );
 
 
+#############################################################################
+##
+#M  OrbitsMovedPoints( <G> )
+##
+InstallMethod(OrbitsMovedPoints,"for a permutation group",true,[IsPermGroup],0,
+function(G)
+local o;
+  o:=Orbits(G,MovedPoints(G));
+  o:=List(o,x->Immutable(Set(x)));
+  Sort(o);
+  return o;
+end);
+
+
+
+
+#############################################################################
+##
+#M  \=( <G>, <H> )  . .  . . . . . . . . .  test if two perm groups are equal
+##
+InstallMethod( \=, "perm groups", IsIdenticalObj, [ IsPermGroup, IsPermGroup ],
+function ( G, H )
+  # avoid new stabilizer chains if they are not computed yet
```

Comment:
If you want another quick pre-test, to avoid even calculating `OrbitsMovedPoints`, you could just check the largest moved points of both `G` and `H`. Of course, depending on your application that always be the same.

Reply:
There isn't a current example, but theoretically there could be groups wiuthout generators (defined as stabilizer of a structure).

The application I have is a group acting on subgroups. There the largest point does not distinguish much, but then again its cost is minimal.